Xamarin.Android.sln

```diff
@@ -460,6 +476,7 @@ Global
 		{0DE278D6-000F-4001-BB98-187C0AF58A61} = {E351F97D-EA4F-4E7F-AAA0-8EBB1F2A4A62}
 		{1640725C-4DB8-4D8D-BC96-74E688A06EEF} = {E351F97D-EA4F-4E7F-AAA0-8EBB1F2A4A62}
 		{7CE69551-BD73-4726-ACAA-AAF89C84BAF8} = {E351F97D-EA4F-4E7F-AAA0-8EBB1F2A4A62}
+		{15945D4B-FF56-4BCC-B598-2718D199DD08} = {04E3E11E-B47D-4599-8AFC-50515A95E715}
```

Comment:
Perhaps this should go into the `Tools` folder? Not sure it's immediately relevant for `Core`.

`Tools` is `864062D3-A415-4A6F-9324-5820237BA058`.